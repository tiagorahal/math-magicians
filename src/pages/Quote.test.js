import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Quote from './quote';

it('matches the snapshot', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders the quote and the author\'s name', () => {
  render(<Quote />);
  const quoteElement = screen.getByText(/William Paul Thurston/i);

  expect(quoteElement).toBeInTheDocument();
});
