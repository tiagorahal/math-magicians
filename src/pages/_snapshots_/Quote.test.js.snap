// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="quote-container"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </h3>
</div>
`;